### ISSUE TITLE
bug: Nerd-Tree Git / Buffers can't show help and Y mapping to unavailable `copy_file_name`

### ISSUE THREAD

**Issue by ytakio** [LazyVim/LazyVim#2510]
### Did you check docs and existing issues?

- [X] I have read all the LazyVim docs
- [X] I have searched the existing issues of LazyVim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

0.9.5

### Operating system/version

Ubuntu 20.04 (WSL2)

### Describe the bug

When Nerd-Tree Git / Buffers are about to show help (? key), these exception is shown.

```bash
E5108: Error executing lua: ...io/.local/share/nvim/lazy/nui.nvim/lua/nui/line/init.lua:22: attempt to index local 'block' (a nil value)
stack traceback:
	...io/.local/share/nvim/lazy/nui.nvim/lua/nui/line/init.lua:22: in function 'append'
	.../lazy/neo-tree.nvim/lua/neo-tree/sources/common/help.lua:62: in function 'show'
	...y/neo-tree.nvim/lua/neo-tree/sources/common/commands.lua:882: in function <...y/neo-tree.nvim/lua/neo-tree/sources/common/commands.lua:879>
```

And these warn that Y mapping to invalid function (it seems `copy_file_name`)
![image](https://github.com/LazyVim/LazyVim/assets/1217667/54960ad5-cf68-45f6-b7bb-e3dde09fcd5a)

### Steps To Reproduce

Open Nerd-Tree Git / Buffers and press '?' key to show help window.

### Expected Behavior

showing help window at Nerd-Tree Git / Buffers withoug exceptions.

### Repro

```Lua
-- DO NOT change the paths and don't remove the colorscheme
local root = vim.fn.fnamemodify("./.repro", ":p")

-- set stdpaths to use .repro
for _, name in ipairs({ "config", "data", "state", "cache" }) do
  vim.env[("XDG_%s_HOME"):format(name:upper())] = root .. "/" .. name
end

-- bootstrap lazy
local lazypath = root .. "/plugins/lazy.nvim"
if not vim.loop.fs_stat(lazypath) then
  vim.fn.system({ "git", "clone", "--filter=blob:none", "https://github.com/folke/lazy.nvim.git", lazypath, })
end
vim.opt.runtimepath:prepend(lazypath)

-- install plugins
local plugins = {
  "folke/tokyonight.nvim",
  "folke/LazyVim",
  -- add any other plugins here
}
require("lazy").setup(plugins, {
  root = root .. "/plugins",
})

vim.cmd.colorscheme("tokyonight")
-- add anything else here
```

**Comment by ytakio** [comment]
Sorry for overlooking latest one.
Its' already fixed.

**Comment by hisea** [comment]
I can still see this error.

**Comment by ytakio** [comment]
@hisea 
I found eccf3b5e68035516c879b3e2b4d6bf8512c87976 this commit has already fixed this bug.
`copy_file_name` function were scoped out before this patch, that's why Nerd-Tree Git/Buffers restulted error while keymapping.

**Comment by hisea** [comment]
Ah, thanks, I think it was merged into main after v10.9.1, which is the latest release version. I will wait for the next release.

**Comment by dpetka2001** [comment]
You could also just choose to follow LazyVim HEAD instead of the latest stable release as described [here](https://www.lazyvim.org/configuration/lazy.nvim).
